Source/SwiftLintFramework/Extensions/File+Cache.swift

```diff
@@ -91,8 +93,8 @@ private class Cache<T> {
 }
 
 extension File {
-    fileprivate var cacheKey: String {
-        return path ?? contents
+    fileprivate var cacheKey: CacheKey {
+        return ObjectIdentifier(self).hashValue
```

Comment:
This has been changed from `ObjectIdentifier` in https://github.com/realm/SwiftLint/pull/837 because there was a problem using `ObjectIdentifier` as Key.

Reply:
Thanks.

Okay. there's one not-that small solution :) like this.

https://github.com/realm/SwiftLint/blob/speedup/test-use-cache-as-int/Source/SwiftLintFramework/Models/SwiftLintFile.swift#L10

Reply:
@norio-nomura so is it only because of tests?

Reply:
Oh they're crashing... then we'll go with the complex solution

Reply:
If an operation that releases a `File` occurs while the` Process` is alive, it may also occur when `swiftlint` is executed. I don't know how long the `File` instances have survived in recent SwiftLint, so I don't know if it will happen in` master` either.

Reply:
Got it. My idea is to create `SwiftLintFile` wrapper over `File` which will have unique id se up on creation. Time spent on simple `Cache.get(key: contents)` is enormous

Reply:
> https://github.com/realm/SwiftLint/blob/speedup/test-use-cache-as-int/Source/SwiftLintFramework/Models/SwiftLintFile.swift#L10

Since `File` instances created from the same file on the file system are required to return the same `cacheKey`, I think using `id` is not suitable for the purpose.